internal/cloudapi/v2/v2_test.go

```diff
@@ -85,6 +85,7 @@ func TestUnknownRoute(t *testing.T) {
 		"id": "21",
 		"kind": "Error",
 		"code": "IMAGE-BUILDER-COMPOSER-21",
+		"details": "",
```

Comment:
Just add it to ignored fields like 'operation_id' below. 'details' shouldn't have any schema attached to it.